mapping_mvp/initial_data/tables_app_data.json

```diff
@@ -1 +1,135 @@
+[
```

Comment:
I don't know why Github is showing a load of whitespace at the top - if you open the file, there's no whitespace.

Reply:
Or is Github simply showing whitespace where old data was lurking?